template/work-item-comment-created/work-item-comment-created.html

```diff
@@ -31,7 +35,7 @@
           </td>
           <td class="table-content" style="text-align: left;">
             <a href="#" style="color: #0088ce; text-decoration: none;">
-              <img src="profile_thumbnail.png" />Adam J. Jolicoeur
```

Comment:
Don't have to fix it in the templates. I've moved them over to a slightly different format